pkgs/tools/text/zimreader/tntnet221.patch

```diff
@@ -0,0 +1,62 @@
+--- a/src/main.cpp	2016-08-17 00:54:46.329502545 +0200
```

Comment:
Oops. Yes, you're right: I've changed the url to a forked repo which is already patched so the the patch is not required anymore here. 
Fixed. Thanks.